### ISSUE TITLE
display command shows all images to be removed ☠️

### ISSUE THREAD

**Issue by dburgos** [ghostboard/ghost-purge-images#7]
ghost-purge-images v3.0.3
Ghost version: v3.6
Reported by @Caucasus-and-Mercury
Related issue #6 

`display` command shows all uploaded images to be removed
I can't reproduce it with Ghost 3.3.0 & ghost-purge-images 3.0.3
Need to investigate, the issue may be related to the API endpoint

```
- /var/www/ghost/content/images/2019/04/siberiannationalcostumes_yakutiarussia_01.jpg (0.09 MB)
- /var/www/ghost/content/images/2019/04/vladimir-kolodenko---aeroport.JPG (0.04 MB)

📊 Summary:
- 2771 files of 2771 uploaded images (100.00%)
- Total space: 969.57MB

❔ Want to delete this files? Run `ghost-purge-images purge key=YOUR_KEY`
🎁 Open source tool by David Burgos from https://ghostboard.io
ghost-mgr@ghost-caucasusandmercury:/var/www/ghost$ 
```

**Comment by Maxszik** [comment]
Hello there @dburgos ,

I'm still having these issues, purging images doesn't work as it used to earlier. 

When running the purge command many, many images are purged that shouldn't be, images that are still in use.

I'm on the digitalocean droplet, latest Ghost 3.15.3. I've been trying to solve things by running the install in the droplet directly as well as in the ghost installation folder. I've also done the `sudo chmod -R a+rw content` here and there. Maybe I've messed up things somehow?

When I do the install it works but I get this:

```
root@ghost-caucasusandmercury:~# cd /var/www/ghost
root@ghost-caucasusandmercury:/var/www/ghost# npm install -g ghost-purge-images
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
/usr/bin/ghost-purge-images -> /usr/lib/node_modules/ghost-purge-images/bin/ghost-purge-images.js
+ ghost-purge-images@3.0.3
updated 4 packages in 3.659s
root@ghost-caucasusandmercury:/var/www/ghost# 
```

And when I do the purge check I get this:

```
- /var/www/ghost/content/images/size/w300/2019/03/tungus-horses.jpg (0.01 MB)
- /var/www/ghost/content/images/size/w300/2019/04/people-of-tartary.jpg (0.02 MB)
- /var/www/ghost/content/images/size/w300/2019/06/2.png (0.09 MB)
- /var/www/ghost/content/images/size/w300/2019/06/i-Jq9qB58-X3.jpg (0.01 MB)
📊 Summary:
- 3242 files of 3242 uploaded images (100.00%)
- Total space: 1539.44MB
❔ Want to delete this files? Run `ghost-purge-images purge key=YOUR_KEY`
🎁 Open source tool by David Burgos from https://ghostboard.io
root@ghost-caucasusandmercury:/var/www/ghost# 
```

Pls note, you made a note yourself above: _display command shows all uploaded images to be removed_. That's true: but all images are not deleted. Not all - but way too many.

Yours is an incredibly useful tool, and I really hope I can make this work somehow. Thank you very much, - Max !

**Comment by dburgos** [comment]
hey @Caucasus-and-Mercury 

Just [released v3.0.4 version](https://www.npmjs.com/package/ghost-purge-images/v/3.0.4) to fix this issue

Thanks for your your patience
